hellabot.go

```diff
@@ -115,9 +115,10 @@ func (bot *Bot) handleIncomingMessages() {
 		bot.con.SetDeadline(time.Now().Add(bot.PingTimeout))
 		msg := ParseMessage(scan.Text())
 		bot.Debug("Incoming", "raw", scan.Text(), "msg.To", msg.To, "msg.From", msg.From, "msg.Params", msg.Params, "msg.Trailing", msg.Trailing)
+		// Otherwise this is mutable by DropTrigger and AddTrigger
```

Comment:
Given that there's no need for a mutex and DropTrigger no longer exists, this comment can be dropped.